examples/webaudio/Cargo.toml

```diff
@@ -16,8 +16,6 @@ features = [
   'AudioDestinationNode',
   'AudioNode',
   'AudioParam',
-  'AudioScheduledSourceNode',
-  'BaseAudioContext',
```

Comment:
Isn't removing features a breaking change?

Reply:
Oh these aren't removed from `web-sys` itself, just here because the features are no longer needed

Reply:
D'oh I totally misread what file this was